### PR TITLE
[MRG+1] Add Code of Conduct Version 1.3.0 from http://contributor-covenant.org/

### LINKED ISSUE #1645: COC?
Hi,

I'm just wondering what your code of conduct is for this project?

Thanks,
Alison

### PR DESCRIPTION
**Pull request by redapple**
Closes #1645

### CODE REVIEW

**Comment by eliasdorneles** [comment]
:+1:

File: README.rst [hunk 1]
```diff
@@ -73,6 +73,12 @@ See http://scrapy.org/community/
 Contributing
 ============
 
+Please note that this project is released with a Contributor Code of Conduct
+(see CODE_OF_CONDUCT.md).
+
+By participating in this project you agree to abide by its terms.
+Please report unacceptable behavior to opensource@scrapinghub.com.
```
Comment (kmike) [inline README.rst]: how hard would it be to setup `..@scrapy` emails?

Comment (eliasdorneles) [inline README.rst]: +1

**Comment by dangra** [comment]
I think we should merge now as-is and change once `...@scrapy.org` emails are ready.

**Comment by eliasdorneles** [comment]
I agree, no need for this to be blocked.

File: README.rst [hunk 2]
```diff
@@ -73,6 +73,12 @@ See http://scrapy.org/community/
 Contributing
 ============
 
+Please note that this project is released with a Contributor Code of Conduct
+(see CODE_OF_CONDUCT.md).
```
Comment (kmike) [inline README.rst]: I think it should be a link to COC at scrapy github master.

**Pull request merged**